### PR TITLE
Author deferred fog component reference page

### PR DESCRIPTION
**Pull request by chanmosq**
## Change summary

- Elaborated introduction
- Added image of feature 
- Filled in properties

@Adi-Amazon - I added you as reviewer of this doc as owner of the feature. 
@galibzon - For awareness. 

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by galibzon — APPROVED** [review]

**Review by hershey5045 — APPROVED** [review]

File: content/docs/user-guide/components/reference/atom/deferred-fog.md [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
 linktitle: Deferred Fog
-description: "Open 3D Engine (O3DE) Deferred Fog component reference."
+description: Create a scene fog or layered/ground fog effect with Deferred Fog component in Open 3D Engine (O3DE).
 toc: true
 ---
 
-The **Deferred Fog** component creates a screen space fog effect that can be used as scene fog or layered/ground fog. You can also add an optional cloud noise turbulence.
+The **Deferred Fog** component creates a screen space fog effect that can be used as scene fog or layered/ground fog. You can also add an optional *cloud turbulence* using a noise texture.
 
+Cloud turbulence is implemented by ray marching along the fog layer and combining two moving noise octaves, creating a "cloudy" fog look. You can configure each octave by scaling the UV coordinates of the noise texture and defining the velocity of the texture's movement. Then, specify the amount of blend between the two octaves.
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/deferred-fog.md:10]:
```suggestion
Cloud turbulence is implemented by ray marching along the fog layer and combining two moving noise octaves, creating a "cloudy" fog look. You can configure each octave by scaling the UV coordinates of the noise texture and defining the velocity of the texture's movement. You can also specify the amount of blend between the two octaves.
```

File: content/docs/user-guide/components/reference/atom/deferred-fog.md [hunk 2]
```diff
@@ -1,20 +1,44 @@
...
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
\ No newline at end of file
+
+![Deferred Fog component interface](/images/user-guide/components/reference/atom/deferred-fog/deferred-fog-component-ui.png)
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Fog Color** | The color of the fog that gradually blends with the scene. Becomes more opaque as the fog blends towards the **Fog End Distance**.  | Color | R: `115`, G: `115`, B: `153` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/deferred-fog.md:31]:
```suggestion
| **Fog Color** | The color of the fog that gradually blends with the scene. The color becomes more opaque as the fog blends towards the **Fog End Distance**.  | Color | R: `115`, G: `115`, B: `153` |
```

File: content/docs/user-guide/components/reference/atom/deferred-fog.md [hunk 3]
```diff
@@ -1,20 +1,44 @@
...
+| **Fog Start Distance** | The distance from the viewer, measured in meter units, at which the fog starts. At this point, the fog begins to gradually blend with the scene until it completely masks the background scene at the **Fog End Distance**. | Float: 0.0 - 5000.0 | `1.0` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/deferred-fog.md:32]:
```suggestion
| **Fog Start Distance** | The distance from the viewer, measured in meters, at which the fog starts. At this distance, the fog begins to gradually blend with the scene until it completely masks the background scene at the **Fog End Distance**. | Float: 0.0 - 5000.0 | `1.0` |
```

File: content/docs/user-guide/components/reference/atom/deferred-fog.md [hunk 4]
```diff
@@ -1,20 +1,44 @@
...
+| **Fog End Distance** | The distance from the viewer, measured in meter units, at which the fog completely masks out the background scene. From the **Fog Start Distance** to the **Fog End Distance**, the fog blends with the scene, creating a gradual transition between the scene and the masked layer of fog. At and beyond this point, the scene appears a solid color, completely masked by the fog layer.  | Float: 0.0 - 5000.0 | `5.0` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/deferred-fog.md:33]:
```suggestion
| **Fog End Distance** | The distance from the viewer, measured in meters, at which the fog completely masks out the background scene. From the **Fog Start Distance** to the **Fog End Distance**, the fog blends with the scene, creating a gradual transition between the scene and the masked layer of fog. At and beyond this distance, the scene appears a solid color, completely masked by the fog layer.  | Float: 0.0 - 5000.0 | `5.0` |
```

File: content/docs/user-guide/components/reference/atom/deferred-fog.md [hunk 5]
```diff
@@ -1,20 +1,44 @@
...
+| **Fog Bottom Height** | The height of the bottom of the fog layer, measured in meter units, when **Enable Fog Layer** is activated. | Float: -5000.0 - 5000.0 | `0.01` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/deferred-fog.md:34]:
```suggestion
| **Fog Bottom Height** | The height of the bottom of the fog layer, measured in meters, when **Enable Fog Layer** is activated. | Float: -5000.0 - 5000.0 | `0.01` |
```

File: content/docs/user-guide/components/reference/atom/deferred-fog.md [hunk 6]
```diff
@@ -1,20 +1,44 @@
...
+| **Fog Max Height** | The height of the top of fog layer, measured in meter units, when **Enable Fog Layer** is activated. | Float: -5000.0 - 5000.0 | `1.0` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/deferred-fog.md:35]:
```suggestion
| **Fog Max Height** | The height of the top of the fog layer, measured in meters, when **Enable Fog Layer** is activated. | Float: -5000.0 - 5000.0 | `1.0` |
```

File: content/docs/user-guide/components/reference/atom/deferred-fog.md [hunk 7]
```diff
@@ -1,20 +1,44 @@
...
+| **Noise Texture** | A single-channel, noise texture that defines the appearance of the fog turbulence, when **Enable Turbulence Properties** is activated. | A `.streamingasset` product asset.  | `textures/cloudnoise_01.jpg.streamingimage` |
+| **Noise Texture First Octave** | Scales the UV coordinates of the **Noise Texture** for the first octave. | Vector2: -Infinity to Infinity | X: `0.01`, Y: `0.01` |
+| **Noise Texture First Octave Velocity** | The velocity of the noise texture's movement for the first octave, measured in meter units. | Vector2: -Infinity to Infinity | X: `0.002`, Y:`0.0032` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/deferred-fog.md:38]:
```suggestion
| **Noise Texture First Octave Velocity** | The velocity of the noise texture's movement for the first octave, measured in meters per second. | Vector2: -Infinity to Infinity | X: `0.002`, Y:`0.0032` |
```

File: content/docs/user-guide/components/reference/atom/deferred-fog.md [hunk 8]
```diff
@@ -1,20 +1,44 @@
...
+| **Noise Texture Second Octave** | Scales the UV coordinates of the **Noise Texture** for the second octave. | Vector2: -Infinity to Infinity | X: `0.0239`, Y: `0.0239` |
+| **Noise Texture Second Octave Velocity** | The velocity of the noise texture's movement for the second octave, measured in meter units. | Vector2: -Infinity to Infinity | X: `0.00275`, Y: `-0.004` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/deferred-fog.md:40]:
```suggestion
| **Noise Texture Second Octave Velocity** | The velocity of the noise texture's movement for the second octave, measured in meters per second. | Vector2: -Infinity to Infinity | X: `0.00275`, Y: `-0.004` |
```

**Review by FiniteStateGit — CHANGES_REQUESTED** [review]
Minor suggestions, feel free to disregard what you don't like!

**Review by Adi-Amazon — APPROVED** [review]
Thanks for adding this documentation.
Perhaps important to add that currently the fog does not interacts with lighting, something that can be next future feature to add and will give the final touch towards being visualized as volumetric fog.

**Comment by chanmosq** [comment]
> Thanks for adding this documentation. Perhaps important to add that currently the fog does not interacts with lighting, something that can be next future feature to add and will give the final touch towards being visualized as volumetric fog.

Added! Thank you for the additional information.

**Review by Adi-Amazon — APPROVED** [review]

**Comment by chanmosq** [comment]
Hey @FiniteStateGit, when you get the chance can you review the changes I made per your suggestions? Thanks!

**Review by FiniteStateGit — APPROVED** [review]
Looks good!

**Pull request merged**